server/jwt/jwt_util.py

```diff
@@ -0,0 +1,24 @@
+import logging, jwt
+from flask import request, jsonify
+from functools import wraps
+from server import app
+from server.models import User
+
+def token_required(fun):
+    @wraps(fun)# proveri zasto se stavlja wraps. zato sto moze da ne vrati objekat
```

Comment:
Wraps is needed to replace your function with a custom one, see this for reference: https://towardsdatascience.com/why-you-should-wrap-decorators-in-python-5ac3676835f9#:~:text=Decorators%20are%20an%20important%20feature,to%2Dbe%2Ddecorated%20function.

Also, you are missing a new line.